common/js/src/popup-window/popup-opener.js

```diff
@@ -70,7 +70,7 @@ const logger = GSC.Logging.getScopedLogger('PopupWindow.PopupOpener');
 /**
  * ID used to differentiate between different popup instances.
  */
-let lastUsedPopupId = 0;
```

Comment:
This variable should actually stay "lastUsedPopupId". My comment was about renaming in in-popup-main-script.js.

To reiterate, the point was that the opener might open multiple popups, and its variable "lastUsedPopupId" is a counter; meanwhile the in-popup-script only stores its own id, which is why "currentPopupId" is better there.